whatsnew/build.gradle

```diff
@@ -37,11 +37,12 @@ android {
 
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
-    implementation "androidx.appcompat:appcompat:${libraries.supportLibraryVersion}"
-    implementation "androidx.recyclerview:recyclerview:${libraries.supportLibraryVersion}"
+    implementation "androidx.appcompat:appcompat:1.0.2"
```

Comment:
here too.

Reply:
Unsure about these ones. AndroidX libraries aren't necessary always the same versions like the support library was.
E.g.
implementation "androidx.appcompat:appcompat:1.0.2"
implementation "androidx.recyclerview:recyclerview:1.0.0"

Reply:
Got that. Creating a new version name (eg recyclerview) is fine.

Reply:
Done :)